QuasarHelios2015/src/org/team1540/quasarhelios/ControlInterface.java

```diff
@@ -85,8 +77,23 @@ private static void setupRollers() {
 
         FloatInput cutoffAuto = mainTuning.getFloat("Trigger Threshold +M", 0.5f);
 
-        BooleanMixing.pumpWhen(QuasarHelios.manualControl, FloatMixing.floatIsAtLeast(Igneous.joystick2.getAxisSource(3), cutoffAuto), QuasarHelios.autoEjector);
-        BooleanMixing.pumpWhen(QuasarHelios.manualControl, FloatMixing.floatIsAtLeast(Igneous.joystick2.getAxisSource(4), cutoffAuto), QuasarHelios.autoLoader);
+        FloatInput leftTrigger = Igneous.joystick2.getAxisSource(3);
+        FloatInput rightTrigger = Igneous.joystick2.getAxisSource(4);
+
+        BooleanInputPoll rollersMode = Igneous.joystick2.getButtonChannel(5);
```

Comment:
Maybe there's a button on the front somewhere that we could exchange with one of the bumpers? It seems easier to hold one thing with your thumb and work with the triggers than holding one of the bumpers.